funnel/assets/js/vue_util.js

```diff
@@ -64,7 +64,7 @@ export const faSvg = Vue.component('faicon', {
 });
 
 export const shareDropdown = Vue.component('sharedropdown', {
-  template: `<ul class="mui-dropdown__menu mui-dropdown__menu--hg-link" data-cy="share-dropdown"><li><a class="mui--text-body2 js-copy-link" href="javascript:void(0)" data-action="Copy link"><faicon :icon="'copy'" :baseline=true :css_class="'mui--text-light fa-icon--margin'"></faicon>Copy link<span class="js-copy-url">{{ url }}</span></a></li><li><a class="mui--text-body2" :href="emailUrl" data-action="Share via Email"><faicon :icon="'envelope'" :baseline=true :css_class="'mui--text-light fa-icon--margin'"></faicon>Email</a></li><li><a class="mui--text-body2" target="_blank" rel="noopener" :href="twitterUrl" :data-url="url" data-via="Hasgeek" :data-text="title" data-action="Tweet"><faicon :icon="'twitter-square'" :baseline=true :css_class="'mui--text-light fa-icon--margin'"></faicon>Twitter</a></li><li><a class="mui--text-body2" target="_blank" rel="noopener" :href="facebookUrl" :data-href="url" data-action="Share on facebook"><faicon :icon="'facebook-square'" :baseline=true :css_class="'mui--text-light fa-icon--margin'"></faicon>Facebook</a></li><li><a class="mui--text-body2" :href="linkedinUrl" data-action="Share about on linkedin"><faicon :icon="'linkedin'" :baseline=true :css_class="'mui--text-light fa-icon--margin'"></faicon>Linkedin</a></li><ul>`,
+  template: `<ul class="mui-dropdown__menu mui-dropdown__menu--hg-link" data-cy="share-dropdown"><li><a class="mui--text-body2 js-copy-link" href="javascript:void(0)" data-ga="Copy link"><faicon :icon="'copy'" :baseline=true :css_class="'mui--text-light fa-icon--margin'"></faicon>Copy link<span class="js-copy-url">{{ url }}</span></a></li><li><a class="mui--text-body2" :href="emailUrl" data-ga="Share via Email"><faicon :icon="'envelope'" :baseline=true :css_class="'mui--text-light fa-icon--margin'"></faicon>Email</a></li><li><a class="mui--text-body2" target="_blank" rel="noopener" :href="twitterUrl" :data-url="url" data-via="Hasgeek" :data-text="title" data-ga="Tweet"><faicon :icon="'twitter-square'" :baseline=true :css_class="'mui--text-light fa-icon--margin'"></faicon>Twitter</a></li><li><a class="mui--text-body2" target="_blank" rel="noopener" :href="facebookUrl" :data-href="url" data-ga="Share on facebook"><faicon :icon="'facebook-square'" :baseline=true :css_class="'mui--text-light fa-icon--margin'"></faicon>Facebook</a></li><li><a class="mui--text-body2" :href="linkedinUrl" data-ga="Share about on linkedin"><faicon :icon="'linkedin'" :baseline=true :css_class="'mui--text-light fa-icon--margin'"></faicon>Linkedin</a></li><ul>`,
```

Comment:
Won't this render the whole URL to screen readers? They shouldn't read the URL:

`Copy link<span class="js-copy-url">{{ url }}</span>`